run/identify-sha.sh

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 The WPT Dashboard Project. All rights reserved.
+
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file.
+
+# Run this script in cron, and have it write to the
+# /var/www/html folder a file called sha
+#
+# You should have Options +Includes, mod_include and XBitHack on in
+# your <VirtualHost> for the htdocs/index.html page to work.
+
+sha="$(git ls-remote https://github.com/w3c/web-platform-tests.git | head -1 | cut -f 1)"
```

Comment:
I guess this is a placeholder? I think that for external runs, right now looking for the latest run in https://wpt.fyi/test-runs with "many" browsers would be a better bet, but an API that just tells you what to try running and in which order would be nicer.

Reply:
This would be where runners would look to populate that test-runs page. 

Every browser could `curl https://sha.wpt.fyi/sha` and get a SHA and pass that to run.py or whatever they're using.

Reply:
The challenge with external runs ("outside of the wpt.fyi infrastructure") is how to test the same commit of wpt as has already been tested on wpt.fyi, or is in the process of being tested. https://github.com/w3c/wptdashboard/issues/292 has some links to related issues.

If one is OK with always being behind, https://wpt.fyi/api/runs?max-count=100 is a decent API for this already. To improve upon that, I think that the commit to be tested needs to be published while it is being tested. Then one could either use that and https://wpt.fyi/api/runs?max-count=100 to determine which commits to test, or a single feed of commits to test could be provided. If it's a single feed, then possible some notion of priority is needed, so that external runs that runs just daily/weekly/monthly will tend to use the same commit.